src/index-router/index-withdraw-router.ts

```diff
@@ -12,15 +12,18 @@ import { IIndexRouter } from '../typechain/IndexRouter'
 import { Address, ChainId, ChainIds, isAddress } from '../types'
 
 /** ### Default IndexRouter address for network */
-export const defaultIndexRouterAddress: Record<ChainId, Address> = {
+export const defaultIndexWithdrawRouterAddress: Record<ChainId, Address> = {
   /** ### Default IndexRouter address on mainnet. */
   [ChainIds.Mainnet]: '0x1985426d77c431fc95e5ca51547bcb9b793e8482',
   /** ### Default IndexRouter address on c-chain. */
-  [ChainIds.CChain]: '0xd6dd95610fc3a3579a2c32fe06158d8bfb8f4ee9',
+  // [ChainIds.CChain]: '0xd6dd95610fc3a3579a2c32fe06158d8bfb8f4ee9',//INFO: main
+  [ChainIds.CChain]: '0x848c8b8b1490e9799dbe4fe227545f33c0456e08', //INFO: staging
```

Comment:
Hmm we have alot of magic variables floating around. Maybe we should create a single const file to store these addresses (incl their staging). I also noticed it [here](https://github.com/Phuture-Finance/phuture-sdk/blob/0d2a2a409659d4161939e5abd14f5081a26bfae3/src/price-oracle/phuture-price-oracle.ts#L15)